### ISSUE TITLE
Title is only needed when a publisher has multiple programmes?

### ISSUE THREAD

**Issue by stevieflow** [ThreeSixtyGiving/registry#27]
As we will display the grant date range in the Registry, this *starts* to illustrate that the need to manually keep maintain that in the title is largely superfluous.  It's also often the case that this text becomes out of synch with the actual data - therefore we could guide publishers accordingly:

![data registry 360giving](https://user-images.githubusercontent.com/1209944/43833772-c913ea8c-9b03-11e8-8eec-a02422d4aa71.png)

This would be acceptable to publishers who publisher file(s) that are time bound.  There is an argument that we could *remove* the Title field from the initial registry table view.  This would certainly provide a better data experience for people, with non standard narratives removed

Our difficulty comes for those publishers who segment their data via programmes, for example:

![data registry 360giving 1](https://user-images.githubusercontent.com/1209944/43833838-ff746a7a-9b03-11e8-82bb-6825c875b9f8.png)

In this instance, removing the titles would also remove any clarity for users.

A _possible_ way around this could be a new metadata field for programmatic "coverage" or "scope", whereby the publisher could indicate (with guidance) what the file covers -- > All grants or "Programme X", for example.

(but that might become just another version of the Title field!)

For discussion - pinging @morchickit @KDuerden

**Comment by stevieflow** [comment]
Thanks - understood.  If @KDuerden can change the *existing* titles it'd be very helpful to our registry view.  But - I'm aware that this might not be the whole solution - especially if publishers start to output their own metadata.  

The whole of this might be best sorted through guidance though - rather than another new field.  That should be the last resort

**Comment by KDuerden** [comment]
Yes. I can amend titles to make them fit and be more consistent but I wasn't going to make any changes until the registry project is a bit further along. 
It would be useful to have some naming conventions to use, especially if the longer term aim is for the title field to be pulled from metadata, because this could form part of our guidance.

**Comment by stevieflow** [comment]
This doesnt seem to be a technical issue.  We'll include reference to this in the feedback , as a protocol for 360 to look at
